### ISSUE TITLE
[Arm64] Crossgen of ConsoleMandel.exe fails

### ISSUE THREAD

**Issue by sdmaclea** [dotnet/coreclr#16541]
```
$PWD/Tests/coreoverlay/crossgen /Platform_Assemblies_Paths $PWD/Tests/coreoverlay:$PWD/Tests/JIT/Performance/CodeQuality/SIMD/ConsoleMandel/ConsoleMandel/IL:$PWD/Tests//JIT/Performance/CodeQuality/SIMD/ConsoleMandel/ConsoleMandel /in ConsoleMandel.org /out ConsoleMandel.exe

Assert failure(PID 33337 [0x00008239], Thread: 33337 [0x8239]): 
Assertion failed 'varDsc->lvFieldCnt == 1' in 'Algorithms.ComplexVecFloat:op_Addition(struct,struct):struct' (IL size 40)

    File: src/jit/lclvars.cpp Line: 1972
```

Test was disable in CI by #16538

**Comment by RussKeldorph** [comment]
Move out if ARM64-specific.

**Comment by RussKeldorph** [comment]
Fixed by #17575
